src/main/java/step3/Car.java

```diff
@@ -0,0 +1,37 @@
+package step3;
+
+public class Car {
+  private int location = 0;
+
+  public Car() {
+  }
+
+  public Car(int location) {
+    this.location = location;
+  }
+  public void moveConditionally(final int condition) {
+    if (meetMoveCondition(condition)) {
+      this.location++;
+    }
+  }
+
+  private boolean meetMoveCondition(final int condition) {
+    final int CRITERIA = 4;
+    return condition >= CRITERIA;
+  }
+
+  public boolean isLocationValueOf(final int location) {
+    return this.location == location;
+  }
+
+  public String trace() {
```

Comment:
>여전히 반환 값을 활용하는 쪽에서 본 의미가 포함된 것 같기도 하네요 ㅠ 

Car 입장에서는 그냥 문자열을 붙이는거 아닐까요? 
```suggestion
  public String blabla(String "-") {
```
호출하는 측면에서의 개념을 분리시키고 싶다면 기능만 남겨두고 인자로 받는다면 어떨까요?

Reply:
아! `Car` 입장에서 `-`이건 `~` 이건 전혀 상관이 없군요. 그냥 호출측에서 던져준 단위 거리 표시자(`-`)를 거리만큼 더해주기면 하면 된다고 생각하면 되네요ㅎㅎ 리뷰 감사합니다!